man/idig_search_records.Rd

```diff
@@ -57,6 +57,30 @@ this:
 \preformatted{
 rq <- list("scientificname"=list("type"="exists"), "family"="asteraceae")
 }
+
+An example of a more complex JSON query
+
+{```
```

Comment:
In the R documentation format, the 3 tick marks, is not understood. You could use `\code{}` instead.